src/applications/vaos/containers/ExpressCareDetailsPage.jsx

```diff
@@ -10,27 +10,24 @@ import {
   EXPRESS_CARE_ERROR_REASON,
 } from '../utils/constants';
 import FormButtons from '../components/FormButtons';
-import ExpressCareReasonField from '../components/ExpressCareReasonField';
+import TextareaWidget from '../components/TextareaWidget';
+import { validateWhiteSpace } from 'platform/forms/validations';
+import { getExpressCareFormPageInfo } from '../utils/selectors';
 
 import * as actions from '../actions/expressCare';
 
-const pageKey = 'form';
-const pageTitle = 'Select a reason for your Express Care request';
+const pageKey = 'details';
+const pageTitle = 'Express Care request details';
 
 const initialSchema = {
   type: 'object',
   properties: {
-    reasonForRequest: {
-      type: 'object',
-      required: ['reason'],
-      properties: {
-        reason: {
-          type: 'string',
-        },
-        additionalInformation: {
-          type: 'string',
-        },
-      },
+    additionalInformation: {
+      title: (
+        <h2 className="vads-u-color--gray-dark">Tell us about your symptom</h2>
+      ),
```

Comment:
Normally this would be set in the uiSchema, but this heading needs to be dynamically changed, which we handle in the `FORM_ADDITIONAL_DETAILS_PAGE_OPENED` reducer.

Reply:
Tested it out and looks like it's not necessary here, but it is necessary in when using in a `ui:title` like below.  There is some style elsewhere that is making `<legend>`s blue.  Also replacing `<h2>` with `<span>` here as per your suggestion below.